force-app/main/default/classes/LeadConversion.cls

```diff
@@ -0,0 +1,68 @@
+public without sharing class LeadConversion {
+  @InvocableMethod(Label='Convert Leads')
```

Comment:
Best practice is to include a `description` in the `@InvocableMethod` decorator: https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_annotation_InvocableMethod.htm